kubernetes_state/check.py

```diff
@@ -93,6 +93,9 @@ def __init__(self, name, init_config, agentConfig, instances=None):
             'kube_replicationcontroller_status_replicas': 'replicationcontroller.replicas',
             'kube_statefulset_replicas': 'statefulset.replicas_desired',
             'kube_statefulset_status_replicas': 'statefulset.replicas',
+            'kube_statefulset_status_replicas_current': 'statefulset.replicas.current',
```

Comment:
For consistency, let's mirror `kube_deployment_*` metric naming: https://github.com/DataDog/integrations-core/pull/936/files#diff-f11fa8f6e957735347a7525b935dc618R58